pps_mw_training/pipelines/cloud_base/training.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+
+from pps_mw_training.models.trainers.unet_trainer import UnetTrainer
+from pps_mw_training.pipelines.cloud_base import evaluation
+from pps_mw_training.pipelines.cloud_base import settings
+from pps_mw_training.pipelines.cloud_base import training_data
+
+
+def train(
+    n_layers: int,
+    n_features: int,
+    training_data_path: Path,
+    train_fraction: float,
+    validation_fraction: float,
+    test_fraction: float,
+    batch_size: int,
+    n_epochs: int,
+    model_config_path: Path,
+    only_evaluate: bool,
+):
+    "Run the cloud base training pipeline."
+    train_ds, val_ds, test_ds = training_data.get_training_dataset(
+        training_data_path,
+        train_fraction,
+        validation_fraction,
+        test_fraction,
+        batch_size,
+        settings.INPUT_PARAMS,
+        settings.LABEL_PARAMS,
+        settings.FILL_VALUE_IMAGES,
+        settings.FILL_VALUE_LABELS,
+    )
+    if not only_evaluate:
+        UnetTrainer.train(
+            settings.INPUT_PARAMS,
+            settings.N_UNET_BASE,
+            settings.N_UNET_BLOCKS,
+            n_features,
+            n_layers,
+            settings.QUANTILES,
+            train_ds,
+            val_ds,
+            n_epochs,
+            settings.FILL_VALUE_IMAGES,
+            settings.FILL_VALUE_LABELS,
+            settings.IMAGE_SIZE,
+            settings.INITIAL_LEARNING_RATE,
+            settings.DECAY_STEPS_FACTOR,
+            settings.ALPHA,
+            settings.AUGMENTATION_TYPE,
+            settings.SUPER_RESOLUTION,
```

Comment:
The inputs are a bit ordered, i.e. the first ones relates to the unet configuration so I think this should be moved up to these.

Reply:
In that case, SUPER_RES should be moved to be with unet configuration but shouldnt AUGMENTATION should be with IMAGE data?

Reply:
you are correct.

Reply:
Fixed